Runtimes/Supplemental/Distributed/CMakeLists.txt

```diff
@@ -143,8 +140,22 @@ target_link_libraries(swiftDistributed PRIVATE
   swift_Concurrency
   swift_Builtin_float
   $<$<PLATFORM_ID:Android>:swiftAndroid>
-  $<$<PLATFORM_ID:Windows>:swiftWinSDK>)
-  # swiftDarwin/Libc/Platform
+  $<$<PLATFORM_ID:Windows>:swiftWinSDK>
+  $<$<PLATFORM_ID:Windows>:swiftDarwin>)
+
+if(TARGET swiftRuntimeCMakeConfig)
```

Comment:
Rather than a separate target, can we make this part of the `swiftCore` `INTERFACE` (through the consumption of `swiftRuntime`)?